src/utils/config.ts

```diff
@@ -0,0 +1,55 @@
+const quirks = !!process.env.QUIRKS;
+
+export type Config = Map<
+  | "cast-objects-in-pact"
+  | "disable-multipart-formdata"
+  | "ignore-duplicate-slashes"
+  | "ignore-trailing-slash"
+  | "legacy-parser"
+  | "no-authorization-schema"
+  | "no-percent-encoding"
+  | "no-transform-non-nullable-response-schema"
+  | "no-validate-complex-parameters"
+  | "no-validate-request-body-unless-application-json",
+  boolean
+>;
+
+export const DEFAULT_CONFIG: Config = new Map([
```

Comment:
What's the plan to use this? Does it allow us to to enable an error and fix it one by one?

Reply:
See my new commit